solutions/gold/usaco-694.mdx

```diff
@@ -165,5 +165,57 @@ public class HoofPaperScissors {
 }
 ```
 </JavaSection>
+<PySection>
 
+<Warning>
+
+Due to Python's constant factor, the below solution TLEs on a couple test cases.
+
+</Warning>
+
+```py
+MX = 10**5 + 5
+
+dp = [[[0] * 3 for _ in range(25)] for _ in range(MX)]
```

Comment:
why can't you declare after input & define with variable sizes?

Reply:
done